app/code/Inchoo/Sample07/Setup/Patch/Data/CreateSeasonProductAttribute.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\Sample07\Setup\Patch\Data;
+
+use Magento\Catalog\Api\Data\ProductAttributeInterface;
+use Magento\Eav\Setup\EavSetupFactory;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+class CreateSeasonProductAttribute implements DataPatchInterface
+{
+    /**
+     * @var EavSetupFactory
+     */
+    protected $eavSetupFactory;
+
+    /**
+     * CreateSeasonProductAttribute constructor.
+     * @param EavSetupFactory $eavSetupFactory
+     */
+    public function __construct(EavSetupFactory $eavSetupFactory)
+    {
+        $this->eavSetupFactory = $eavSetupFactory;
+    }
+
+    /**
+     * @return string[]
+     */
+    public static function getDependencies()
+    {
+        return [];
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getAliases()
+    {
+        return [];
+    }
+
+    /**
+     * @return \Magento\Framework\Setup\Patch\PatchInterface
+     */
+    public function apply()
+    {
+        /** @var \Magento\Eav\Setup\EavSetup $eavSetup */
+        $eavSetup = $this->eavSetupFactory->create();
+
+        $attributeCode = 'season';
+        $entityType = ProductAttributeInterface::ENTITY_TYPE_CODE;
+
+        // create or update attribute
+        $eavSetup->addAttribute($entityType, $attributeCode, [
+            'type'         => 'varchar',
+            'input'        => 'multiselect',
+            'label'        => 'Season',
+            'required'     => 0,
+            'user_defined' => 1, // required for custom attributes
+            'visible' => true,
+            'backend' => 'Magento\Eav\Model\Entity\Attribute\Backend\ArrayBackend',
+            'source' => 'Inchoo\Sample07\Model\Source\Attribute\Season'
```

Comment:
ako napraviš ovako biti će lakše searchati po projektu gdje se koriste te klase

```suggestion
            'backend' => \Magento\Eav\Model\Entity\Attribute\Backend\ArrayBackend::class,
            'source' => \Inchoo\Sample07\Model\Source\Attribute\Season::class
```

Reply:
oke, ti i phpstorm ne možete biti u krivu :)